### ISSUE TITLE
Add option to automatically call `refresh_from_db`

### ISSUE THREAD

**Issue by jonashaag** [vandersonmota/model_mommy#337]
Let's say I'm lazy and specify timestamps as follows:

```
foo = mommy.make('Foo', created='2017-02-01')
```

Now `foo.created == '2017-02-01'`. This may be unhandy if you then want to use the value, for example in assertions:

```
self.assertEqual(foo.created, datetime.datetime(...))
```

**Comment by berinhard** [comment]
@jonashaag maybe we could add flag like `mommy.make('Foo', created='2017-02-01', _refresh_after_create=True)`.

**Comment by berinhard** [comment]
Fixed by #364 .
